Tools/MonoGame.Content.Builder/BuildContent.cs

```diff
@@ -306,7 +306,7 @@ public void Build(out int successCount, out int errorCount)
             // so it can resolve importers, processors, writers, and types.
             foreach (var r in References)
             {
-                var assembly = r;
+                var assembly = ReplaceSymbols(r);
```

Comment:
Seems like a neat trick.

@harry-cpp ?

Reply:
Yea, we are gonna have to add support to mgcb-editor for it to be able to add symbols to reference paths.

Reply:
From the GUI you mean? As far as I understood ticket #7232 it looked like a CLI/manual call but I must admit that I'm fairly new to the project.

Reply:
@harry-cpp can you point @Taronyu at the code for that in the editor?

Reply:
It does not brake anything, just adds it to the CLI, I'll fix the editor when I get the chance.